programs/psa/key_ladder_demo.sh

```diff
@@ -1,16 +1,13 @@
 #!/bin/sh
-set -e -u
+. "${0%/*}/../demo_common.sh"
 
-program="${0%/*}"/key_ladder_demo
-files_to_clean=
+msg <<'EOF'
+This script demonstrates the use of the PSA cryptography interface to
+create a master key, derive a key from it and use that key to wrap
```

Comment:
Minor: perhaps it's just me, but I find "that key" to be a bit ambiguous here, and I'd prefer the repetition: "use the master key".

Reply:
It's the derived key, actually. And it's used to wrap some data, not a key!